java/src/main/java/com/databricks/mlflow/LoaderModuleException.java

```diff
@@ -0,0 +1,7 @@
+package com.databricks.mlflow;
+
+public class LoaderModuleException extends Exception {
```

Comment:
I'm not sure we should rely on checked exceptions, especially a generic checked exception. We should discuss what exceptions should be thrown in each situation.